app/v2/generators/DepartureMessageGenerator.scala

```diff
@@ -266,4 +270,85 @@ class DepartureMessageGeneratorImpl @Inject()(clock: Clock) extends Generators w
         </Consignment>
       </ncts:CC029C>
     )
+
+  private def generateIE056Message(correlationId: String): XMLMessage =
+    XMLMessage(
+      <ncts:CC056C PhaseID="NCTS5.0" xmlns:ncts="http://ncts.dgtaxud.ec">
+        <messageSender>{alphanumeric(35)}</messageSender>
+        <messageRecipient>{correlationId}</messageRecipient>
+        <preparationDateAndTime>{generateLocalDateTime()}</preparationDateAndTime>
+        <messageIdentification>{alphanumeric(35)}</messageIdentification>
+        <messageType>CC056C</messageType>
+        <correlationIdentifier>{correlationId}</correlationIdentifier>
+        <TransitOperation>
+          <LRN>{Strings.alphanumeric(2, 22)}</LRN>
```

Comment:
This should be {alphanumeric(1, 22)}
 <xs:simpleType name="Alphanumeric_Max22">
    <xs:restriction base="AlphaNumType">
      <xs:maxLength value="22" />
      <xs:pattern value=".{1,22}" />
    </xs:restriction>
  </xs:simpleType>